### ISSUE TITLE
Can `--transform` support to pass args ?

### ISSUE THREAD

**Issue by yjhmelody** [AssemblyScript/assemblyscript#1691]
It seems there is no way to pass args to control the behaviors of transform hooks. Has any plan to support it ? Or any other instead way to do it.

**Comment by dcodeIO** [comment]
If there is a need for it, perhaps we can add it. Maybe something along the lines of `--transform path/to/transform?arg1=1&arg2=2`?

**Comment by yjhmelody** [comment]
Further, it may be better if we can get some args passed by asc.

**Comment by jtenner** [comment]
Is it possible to obtain what `--use` flags were passed to the compiler? :)

**Comment by yjhmelody** [comment]
> If there is a need for it, perhaps we can add it. Maybe something along the lines of `--transform path/to/transform?arg1=1&arg2=2`?

It's a good idea. And `transform hook` need to know asc's args.

**Comment by willemneal** [comment]
Yeah I think that passing the CLI args to the transform is all you need.  Since you'd get the extra args unknown to the normal CLI.

**Comment by yjhmelody** [comment]
@dcodeIO Do you have some ideas about this features?
